index.html

```diff
@@ -1,9 +1,26 @@
 ---
 title: home
 layout: default
+my_variable: landing-page-head.html
```

Comment:
Can we have a more descriptive name? I would recommend something like `additional_resources` or `custom_head`

Reply:
ok, changing to 'custom_head'